ray_provider/xcom/ray_backend.py

```diff
@@ -31,11 +32,29 @@ def get_or_create_kv_store(identifier, allow_new=False):
 
 
 class KVStore:
+
```

Comment:
Should we even put this in an xcom folder anymore? Maybe it should exist in a utils folder?